kin_base/horizon.py

```diff
@@ -113,7 +113,6 @@ def __init__(self,
     async def __aexit__(self, exc_type, exc_val, exc_tb) -> None:
         await self.close()
 
-    @_retry
```

Comment:
the bug is that the retry happens in `kin-base` and it's not needed here?

Reply:
This is kin_base

The bug is that _retry happened on "submit" and on "_post", but since "submit" uses "_post", the retry happened 5^2 times

Reply:
yep, silly me, anyway got it